src/main/java/frc/robot/QuickActions.java

```diff
@@ -1,3 +1,5 @@
+// yoinky yoinky ahahahaha this code is mine now yippee!!1!
```

Comment:
im gonna steel the code bc im cooler 😱

Reply:
(this is a test PR guys pls dont accept)

Reply:
ok i will accept it :)